website/docs/routes/new-media.mdx

```diff
@@ -4214,6 +4214,10 @@
 
 <Route author="SunShinenny hoilc" example="/sspai/author/796518" path="/sspai/author/:id" paramsDesc={['作者 slug 或 id，slug 可在作者主页URL中找到，id 不易查找，仅作兼容']} radar="1" />
 
+### 作者文章 {#shao-shu-pai-sspai-zuo-zhe-wen-zhang}
+
+<Route author="SunShinenny hoilc" example="/sspai/author/725653/%E6%9C%AC%E5%91%A8%E7%9C%8B%E4%BB%80%E4%B9%88" path="/sspai/author/:id/:keyword?" paramsDesc={['作者 slug 或 id，slug 可在作者主页URL中找到，id 不易查找，仅作兼容']} radar="1" />
```

Comment:
Please update the previous line instead.

Reply:
done

Reply:
https://github.com/DIYgod/RSSHub/actions/runs/7811606042/job/21307004718?pr=14413
docs: https://docs.rsshub.app/joinus/new-rss/add-docs
Every route parameter should have a description

Reply:
@TonyRL done